packages/webapp/pages/account/invite.tsx

```diff
@@ -68,10 +68,10 @@ const AccountInvitePage = (): ReactElement => {
     },
   );
   const users: UserShortProfile[] = useMemo(() => {
-    const list = [];
+    const list: UserShortProfile[] = [];
     usersResult.data?.pages.forEach((page) => {
       page?.referredUsers.edges.forEach(({ node }) => {
-        list.push(node);
+        list.push(node as UserShortProfile);
       });
     }, []);
```

Comment:
We are forcing node to be a `UserShortProfile` here but we can gentle make it works with something like this:

```ts
const users: UserShortProfile[] = useMemo(() => {
    return usersResult.data?.pages.flatMap((page) =>
      page?.referredUsers?.edges.map(({ node }) => {
        const {
          username,
          id,
          name,
          image,
          bio,
          createdAt,
          reputation,
          permalink,
        } = node;
        return {
          username,
          id,
          name,
          image,
          bio,
          createdAt,
          reputation,
          permalink,
        };
      }),
    );
  }, [usersResult]);
```

It's typesafe and we could create a transform utility from `PublicProfile` to `UserShortProfile` to make it consistent and reusable. What do you think about this approach?